host2str.c

```diff
@@ -1519,6 +1519,30 @@ svcparam_ipv6hint2buffer_str(ldns_buffer *output, size_t sz, uint8_t *data)
 	return ldns_buffer_status(output);
 }
 
+static ldns_status
+svcparam_dohpath2buffer_str(ldns_buffer *output, size_t sz, uint8_t *data)
+{
+	size_t i;
+
+	/* TODO: If the "alpn" SvcParam indicates support for HTTP,
+	 *	"dohpath" MUST be present. */
+
+	/* TODO: The URI Template MUST contain a "?dns" variable  */
+
+	for (i = 0; i < sz; i++) {
+		uint8_t ch = data[i];
+
+		if (isprint(ch) || ch == '\t') {
+			if (ch == '"' ||  ch == ',' || ch == '\\')
+				ldns_buffer_write_char(output, '\\');
+			ldns_buffer_write_char(output, ch);
+		} else
+			ldns_buffer_printf(output, "\\%03u", (unsigned)ch);
+	}
+
+	return ldns_buffer_status(output);
+}
+
```

Comment:
This function is no different than `svcparam_value2buffer_str()` below, so let's use that instead and remove this on.
```suggestion
```